Automattic/wp-calypso
# Site Assembler - Feature flag clean-up

Related #74037 #75326 #74813

## Proposed Changes

* Remove pattern-assembler/notices
* Remove flag pattern-assembler/dotcompatterns
* Remove flag pattern-assembler/inline-styles

## Testing Instructions



- Create a new site with any plan
- Continue until the Design Picker
- Scroll down and click `Start designing`
- Verify that all the assembler screens work as expected
- Verify the following features
 
**Notices should be shown when you add/replace/remove a pattern**

<img width="469" alt="Screenshot 2566-05-22 at 14 09 36" src="https://github.com/Automattic/wp-calypso/assets/1881481/22b46cf6-85e0-417b-b752-de528c7cde95">

**All Dotcom patterns should be shown thanks to request `/rest/v1.1/ptk/patterns/en`**

<img width="1075" alt="Screenshot 2566-05-22 at 14 08 26" src="https://github.com/Automattic/wp-calypso/assets/1881481/e7416daf-0733-4414-8a0c-8ce0d027d316">

**The param `use_inline_styles` is passed as `true` in the request `/block-renderer/settings`** 

<img width="1055" alt="Screenshot 2566-05-22 at 14 02 28" src="https://github.com/Automattic/wp-calypso/assets/1881481/0997c837-6106-4511-8910-85d0f642e167">

## Pre-merge Checklist
